ironfish-rust/src/signal_catcher.rs

```diff
@@ -2,9 +2,18 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+ #[cfg(target_family = "wasm")]
```

Comment:
This whole Rust file won't be usable in WASM environments, so I'd rather skip the compilation of this file altogether rather than tweaking imports.

Besides that, `ffi::c_int` and `libc::c_int` are interchangable. In fact, they're defined in the same way. I would just change everything to `use core::ffi::c_int` and remove the `use libc::c_int`. Same thing for `c_void`.